setup.py

```diff
@@ -29,7 +29,7 @@
     url='https://github.com/awslabs/mxnet-model-server',
     keywords='MXNet Model Server Serving Deep Learning Inference AI',
     packages=pkgs,
-    install_requires=['mxnet>=0.11.0', 'Flask', 'Pillow', 'requests', 'flask-cors', 'psutil', 'jsonschema', 'onnx-mxnet'],
+    install_requires=['mxnet>=0.11.0', 'Flask', 'Pillow', 'requests', 'flask-cors', 'psutil', 'jsonschema', 'onnx-mxnet', 'boto3'],
```

Comment:
Can we please add onnx-mxnet>=0.2 and while we're there, change mxnet to >=1.0 ?

Reply:
Changed.